### ISSUE TITLE
Doesn't work with 'https://www.tmscustomer.com/Account/Register'

### ISSUE THREAD

**Issue by ken14505** [jswanner/DontF-WithPaste#105]
I just went through registering an account at the website of The Money Source (TMS). During the registration process, it was able to block my using Control-V. I then tried using the Context Menu > Paste, but I think I didn't see Paste there.

I think you can start a registration by going to the website above, and entering the following bogus information. There are 5 steps, but it is step 4 that asks for a password. I found that I could enter bogus information in steps 1 to 3, in order to get to step 4. The bogus information I used was a `loan number` of **1234567890**, name of **John Doe**, `email` of **me@myemail.com**. This got me to step 4, and it is this screen that I was unable to paste into the `Password` boxes. 

If that doesn't work, maybe I can try my hand at obtain and providing some diagnostic information to you. If so, please let me know. I'm using Chrome `Version 86.0.4240.198 (Official Build) (64-bit)` on Windows `Version 10.0.17763 Build 17763`.

—Ken

**Comment by jswanner** [comment]
@ken14505, sorry you were having trouble with the extension.

I went through the registration process without activating the extension for the site, I was able to paste into all the fields until I got to the password fields. At that point I needed to activate the extension, afterwards I was able to paste into the password fields, using Mac keyboard shortcut, context menu, and Edit > Paste from the browser's menu.

It looks like they're only interfering with `paste` events which the extension circumvents. I couldn't find anything where they were additionally interfering with the common paste keyboard shortcut key presses.

I would make sure you've properly activated the extension for that site. This extension is not active on all sites by default, as that would cause a degraded browsing experience on sites that do useful things with `paste` events (like turn them into file uploads).
